assets/javascripts/discourse/connectors/bread-crumbs-right/solved-status-filter.hbs

```diff
@@ -1,10 +1,10 @@
 {{#if siteSettings.solved_enabled}}
-  {{combo-box
-    class="solved-status-filter"
-    content=statuses
-    value=status
-    valueProperty="value"
-    onChange=(action "changeStatus")
-    options=(hash caretDownIcon="caret-right" caretUpIcon="caret-down")
-  }}
+  <ComboBox
+    @class="solved-status-filter"
+    @content={{statuses}}
+    @value={{status}}
+    @valueProperty="value"
+    @options={{hash caretDownIcon="caret-right" caretUpIcon="caret-down"}}
+    {{on "change" (action "changeStatus")}}
```

Comment:
I don't think this will work. ComboBox is a select-kit component and its api requires a `onChange` attribute